scripts/build/patches.js

```diff
@@ -79,8 +79,13 @@ const main = async () => {
 
             const filename = path.basename(file);
 
+            // Just for optimization, skip files that are not in the format of
+            // "{filterId}[_optimized|_without_easylist].txt"
             if (!/\d+(_optimized|_without_easylist)?\.txt/.test(filename)) {
-                console.log(`Skipped generating patch for: ${file}`);
+                // Skip printing logs for non ".txt" files as redundant.
```

Comment:
> // Skip printing logs for non ".txt" files as redundant.

Some filters don't have an extension
`@include "https://www.i-dont-care-about-cookies.eu/abp/"`
`@include "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AnnoyancesList"`

Reply:
@Alex-302 This script checks against filenames only in the file system, nothing more, for proper selection of files to which patches should be generated.